Source/Scene/Model.js

```diff
@@ -1148,16 +1201,23 @@ define([
     var nodeQuaternionScratch = new Quaternion();
     var nodeScaleScratch = new Cartesian3();
 
-    function getTransform(node) {
+    function getTransform(node, version) {
         if (defined(node.matrix)) {
             return Matrix4.fromArray(node.matrix);
         }
 
-        var axis = Cartesian3.fromArray(node.rotation, 0, nodeAxisScratch);
+        var rotation = node.rotation;
```

Comment:
Can we do this in gltfDefaults?  E.g., can gltfDefaults basically support all the 0.8 to 1.0 backwards compatibility?